### ISSUE TITLE
permission denied with scons -j4 on network (WINDOWS only)

### ISSUE THREAD

**Issue by bdbaddog** [SCons/scons#1177]
This issue was originally created at: 2005-06-02 11:05:27.
This issue was reported by: `lim_1`.
lim_1 said at 2005-06-02 11:05:27
>Simple Test case attached. You need to run it many 
>time like 20/30 on slowest network drive you have.
> 
>I am just wandering if someone experience something 
>similar.
>Running scons with -j4 option on windows I am 
>randomly getting:
>fatal error C1083: Cannot open source file: 'src\a\t8.cc': 
>Permission denied
>errors from compiler on just generated file by python 
>function builder.
> 
>I do next:
> 
>t1.itt->my python builder->t1.cc->cl.exe compiler->t1.obj
> 
>I have .itt file from which with my own itt builder, which is 
>Python function I create .cpp file. I do close this file.
>Than C++ compiler (cl) called on this just generated file.
> 
>and randomly I am getting this error from compiler about 
>permission denied.
>At the same time I can open and read this file prior 
>cl.exe invocation in the same python function. 
>Looks like within the same thread there is no problem to 
>open this file for read, but when other thread trying to do 
>it even when it's original thread done working I can get 
>this situation.
> 
>I found that it's more reproducible on slow network drive 
>like 1 in 5 runs, but I also was able to create this 
>condition on local drive only once.
> 
>I have created test example which reproduce this error 
>on network drive (Netapp), but rate is lower like 1 in 20 
>than in my full build.
> 
>This happens only on WINDOWS non of my UNIX 
>platform had this problem.

issues@scons said at 2005-06-02 11:05:27
>Converted from SourceForge tracker item 1213694

garyo said at 2007-12-04 05:55:59
>Marking as dup of #<span></span>1721 which has lots more details on what's happening; this
>bug has a nice simple test case though.
> 
>*** This issue has been marked as a duplicate of 1721 ***

stevenknight said at 2007-12-04 07:01:59
>Created an attachment (id=257)
>test case from SourceForge bug tracker

stevenknight attached [parallel.zip](https://github.com/SCons/tigris-issue-attachments/blob/master/1177/257/parallel.zip) at 2008-03-25 11:08:15.
>test case from SourceForge bug tracker

garyo said this issue  is a duplicate of #1721 at 2007-12-04 05:55:59.
